pumping/src/main/kotlin/com/dpm/pumping/auth/config/AuthInterceptor.kt

```diff
@@ -0,0 +1,29 @@
+package com.dpm.pumping.auth.config
+
+import com.dpm.pumping.auth.application.JwtTokenProvider
+import com.dpm.pumping.auth.util.TokenParser
+import jakarta.servlet.http.HttpServletRequest
+import jakarta.servlet.http.HttpServletResponse
+import org.springframework.http.HttpHeaders
+import org.springframework.stereotype.Component
+import org.springframework.web.servlet.HandlerInterceptor
+
+@Component
+class AuthInterceptor(
+    private val jwtTokenProvider: JwtTokenProvider
+) : HandlerInterceptor {
+
+    override fun preHandle(request: HttpServletRequest, response: HttpServletResponse, handler: Any): Boolean {
+        val accessToken: String = TokenParser.extract(request.getHeader(HttpHeaders.AUTHORIZATION))
+        validateAccessToken(accessToken)
+        val payload: String = jwtTokenProvider.getAccessTokenPayload(accessToken)
+        request.setAttribute("PAYLOAD", payload)
```

Comment:
변수명 이거는 어떠실까용 뭘 넣는지 정확했으면 좋겠어서..!! (그냥 의견입니다..)
```suggestion
        val userId: String = jwtTokenProvider.getAccessTokenPayload(accessToken)
        request.setAttribute("USER_ID", userId)
```

Reply:
으음 저는 payload에 나중에 변경이 생기면 user_id아니고 다른 값들을 넣을 수 있을것같아서 payload라고 해놨어요!
지금은 토큰의 payload에 userId만 넣어있지만 추후에 토큰에 user_id뿐만아니라 로그인 타입같은 인자가 추가되면 변경점이 많을 것 같아서요!
변수명만이라도 userId로 바꾸도록 해볼게요! 어떤가요?

Reply:
아아 그렇네요 항상 확장성을 생각하는..👍 변수명만 바꿔주세용